### ISSUE TITLE
To create general diagrams showing main client and server classes and communication between them.

### ISSUE THREAD

**Issue by igrangel** [i40-Tools/AlligatorWorkbench#5]
1. We need to have a diagram representing classes in the client part
2. We need to have a diagram representing classes in the server part
3. We need to have a diagram connecting classes from client to server and vice-versa

Please upload and comments the images here.

**Comment by dinaminsi** [comment]
Here is the class diagram for the server part.
![class diagram - backend](https://user-images.githubusercontent.com/30018710/29663368-4f3eb160-88fd-11e7-89ea-99e4c124b585.jpg)

**Comment by igrangel** [comment]
The Class IndustryStandard should also not be disconnected

**Comment by igrangel** [comment]
For this, 

> 3 - We need to have a diagram connecting classes from client to server and vice-versa
What we actually need is a sequence diagram showing all the connections of the classes between the client and the server side.

**Comment by dinaminsi** [comment]
Yes I missed those two relationships. Here's an updated version for the backend.

![class diagram - backend updated](https://user-images.githubusercontent.com/30018710/29695955-42ab3a24-8978-11e7-9055-5f2ba428d5cd.jpg)

**Comment by dinaminsi** [comment]
Class diagram for the frontend:
![class diagram - frontend](https://user-images.githubusercontent.com/30018710/29711552-e1b15dfc-89c7-11e7-9ba8-d344a392d17f.jpg)

**Comment by dinaminsi** [comment]
Class diagram for frontend:
![class diagram - frontend](https://user-images.githubusercontent.com/30018710/29715797-602edb4c-89db-11e7-93b5-9fe52400ec30.jpg)

**Comment by dinaminsi** [comment]
Sequence diagram:

![sequence diagram](https://user-images.githubusercontent.com/30018710/29717136-a69766f8-89e0-11e7-88ee-afcf357034c2.jpg)
